weblate/templates/translation.html

```diff
@@ -99,6 +96,17 @@
     </ul>
   </li>
   {% endif %}
+  {% if user_can_add_unit %}
+    <li class="dropdown">
+    <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+      {{ object.component.get_add_label }} <span class="caret"></span>
+    </a>
+    <ul class="dropdown-menu">
+      <li><a href="#new-singular" data-toggle="tab">{% trans "Singular" %}</a></li>
+      <li><a href="#new-plural" data-toggle="tab">{% trans "Plural" %}</a></li>
```

Comment:
I think the plural choice should be available only for languages having plural (plural.number > 1). Otherwise, there would be two same forms.

Reply:
I agree, updated the code the new plural translation string. The form for new plural string and those buttons that leads it will be only shown if `plural.number  > 1`.